### PR TITLE
Creation of module for drive errors and add validations in user controller

### PR DESCRIPTION
**Pull request by Petercheks**
1.- Creation of module for show errors in json format.
2.- Add validations in creation of users.

### CODE REVIEW

File: lib/direct_home_api/model/user.ex [hunk 1]
```diff
@@ -55,20 +56,19 @@ defmodule DirectHomeApi.Model.User do
...
         :type
       ])
-      |> unique_constraint([:email])
-      |> unique_constraint([:document])
+      |> unique_constraint([:email])  
+      |> validate_format(:email, ~r/@/)    
       |> unsafe_validate_unique([:email], Repo)
-      |> unsafe_validate_unique([:document], Repo)
+      |> validate_length(:phone, min: 10)
+      |> validate_length(:password, min: 10)
```
Comment (bian2510) [inline lib/direct_home_api/model/user.ex]: I think 8 is a good length to the password

Comment (Petercheks) [reply lib/direct_home_api/model/user.ex]: listo

Comment (Petercheks) [reply lib/direct_home_api/model/user.ex]: The change is ready

**Review by bian2510 — APPROVED** [review]
Good job

**Pull request merged**